Jenkinsfile_CNP

```diff
@@ -35,6 +35,13 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
     ]
 }
 
+// Vars for Kubernetes
+env.IDAM_API_BASE_URI = 'https://preprod-idamapi.reform.hmcts.net:3511'
+env.S2S_BASE_URI = 'http://rpe-service-auth-provider-aat.service.core-compute-aat.internal'
```

Comment:
Can we use string interpolation:

``` 
env.S2S_BASE_URI = "http://rpe-service-auth-provider-${env}.service.core-compute-${env}.internal"
```

Reply:
good shout, I'll give it a go after the show&tell :+1:

Reply:
Nooope, Jenkins didn't like it. Going back to how it was